app/lib/common/gateways.js

```diff
@@ -189,6 +190,20 @@ export const availableGateways = {
         },
         landing: "https://citadel.li/",
         wallet: "https://citadel.li/wallet/"
+    },
+    DEEX: {
+        id: "DEEX",
+        name: "DEEX",
+        baseAPI: deexAPIs,
+        isSimple: true,
+        simpleAssetGateway: false,
+        fixedMemo: {prepend: "dex:", append: ""},
```

Comment:
Please see: fixedMemo in other integrations (RuDEX):
( "prepend_default" for DepositModal and [gateway].DepositRequest )

Reply:
change prepend on prepend_default